cmake-modules/download_test_data.cmake

```diff
@@ -0,0 +1,54 @@
+# this module is used and required by the nlohmann json lib unit tests.
+# It gets the test data
+
+find_package(Git)
+
+set(JSON_TEST_DATA_URL     https://github.com/nlohmann/json_test_data)
+set(JSON_TEST_DATA_VERSION 2.0.0)
```

Comment:
Why 2.0.0 and not 3.0.0? What version of nlohmann do these test versions correspond to? Based on your rationale, add some comments.

Reply:
This module is from nlohmann repo version 3.8. It points to data version 2.0.

I didn't create this file,

Reply:
Gotcha, makes sense.